src/EventTeachingService.h

```diff
@@ -7,29 +7,27 @@
 
 #include "Service.h"
 
-namespace VLCB
+namespace VLCB 
 {
 
 class Configuration;
 
-class EventTeachingService : public Service
-{
+class EventTeachingService : public Service {
```

Comment:
Please move this curly bracket to a new line.

Reply:
Sorry, missed that one!